test/suite/test_oligarch13.py

```diff
@@ -0,0 +1,116 @@
+
+    oligarch_uris = ["oligarch:test_oligarch13a", "oligarch:test_oligarch13b"]
+    other_uris = ["file:test_oligarch13c"]
+
+    disagg_storages = gen_disagg_storages('test_oligarch13', disagg_only = True)
+    scenarios = make_scenarios(disagg_storages)
+
+    # Load the page log extension, which has object storage support
+    def conn_extensions(self, extlist):
+        if os.name == 'nt':
+            extlist.skip_if_missing = True
+        DisaggConfigMixin.conn_extensions(self, extlist)
+
+    # Custom test case setup
+    def early_setup(self):
+        os.mkdir('follower')
+        # Create the home directory for the PALM k/v store, and share it with the follower.
+        os.mkdir('kv_home')
+        os.symlink('../kv_home', 'follower/kv_home', target_is_directory=True)
+
+    # Test more than one table.
+    def test_oligarch13(self):
+        # Create all tables in the leader
+        for uri in self.oligarch_uris + self.other_uris:
+            cfg = self.create_session_config
+            if not uri.startswith('oligarch'):
+                cfg += ',block_manager=disagg,oligarch_log=(enabled=false),log=(enabled=false)'
+            self.session.create(uri, cfg)
+
+        # Create only the oligarch tables in the follower
+        conn_follow = self.wiredtiger_open('follower', self.extensionsConfig() + ',create,' + self.conn_base_config + 'disaggregated=(role="follower")')
+        session_follow = conn_follow.open_session('')
+        for uri in self.oligarch_uris:
+            cfg = self.create_session_config
+            session_follow.create(uri, cfg)
+
+        # Put data to all tables
+        for uri in self.oligarch_uris + self.other_uris:
+            cursor = self.session.open_cursor(uri, None, None)
+            for i in range(self.nitems):
+                cursor[str(i)] = uri
+                if i % 250 == 0:
+                    time.sleep(1)
+            cursor.close()
+
+        time.sleep(1)
+        self.session.checkpoint()
+        time.sleep(1)
+
+        # Check tables in the leader
+        for uri in self.oligarch_uris + self.other_uris:
+            cursor = self.session.open_cursor(uri, None, None)
+            for i in range(self.nitems):
+                self.assertEquals(cursor[str(i)], uri)
+            cursor.close()
+
+        # Pick up the checkpoint in the follower
+        self.disagg_advance_checkpoint(conn_follow)
+
+        # Check tables in the follower
+        for uri in self.oligarch_uris + self.other_uris:
+            # XXX Non-oligarch tables still have transaction IDs from the leader, so we need to set
+            #     the isolation level to "read-uncommitted" to disable the visibility checks.
+            if not uri.startswith('oligarch'):
+                session_follow.begin_transaction('isolation="read-uncommitted"')
+            cursor = session_follow.open_cursor(uri, None, None)
+            for i in range(self.nitems):
+                self.assertEquals(cursor[str(i)], uri)
```

Comment:
nitpick: may be change uri to a different value? It is a bit confusing to compare with the uri.

Reply:
Fixed in [595f67f](https://github.com/wiredtiger/wiredtiger/pull/11211/commits/595f67f13c1042cd5b286e63252c89647b9a6e07).